Documentation/CONFORMANCE_TESTS.md

```diff
@@ -0,0 +1,61 @@
+# Swift Protobuf Conformance Tester
+
+**Welcome to Swift Protobuf!**
+
+Apple's Swift programming language is a perfect complement to Google's Protocol
+Buffer serialization technology.  They both emphasize high performance
+and programmer safety.
+
+Google's protobuf project includes a conformance checking host program that generates a large number of test cases, feeds them to a test program that executes those test cases, then verifies the results.  This project provides the test program for Swift Protobuf that works with Google's conformance checking host to verify that Swift Protobuf is completely compatible with other implementations of Google's protobuf specification.
+
+## Requirements
+
+This test suite requires Swift 3.0, standard command-line tools such as make and awk, and a full source checkout of [Google's protobuf project](https://github.com/google/protobuf).
+
+## Building the Tests
+
+You should start by editing the `Makefile` and adjusting these two lines to match your system:
```

Comment:
you can invoke make with: `make [target] PROTOC=[path] GOOGLE_PROTOBUFS_CHECKOUT=[path]` so no editing is needed.